db/migrate/20211119184322_add_status_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :status, :string, :default => "pending"
```

Comment:
Since the default is 'pending' here, all users will become pending when this goes live (and no longer receive notifications).

I agree the default status should be pending, but perhaps we should add a migrate to set all existing users status to active?

Reply:
Sounds good, migration created!